views.sql

```diff
@@ -0,0 +1,30 @@
+-- The view "get_rented_books_view" provides information about currently rented books.
+-- It displays the title, comma-separated authors, publisher name, release year, loan date in a formatted way,
+-- and the number of days until the end of the term.
+CREATE VIEW get_rented_books_view AS
+    SELECT
+        b.title AS title,
+        GROUP_CONCAT(a.name_ ORDER BY a.name_ ASC SEPARATOR ', ') AS authors,
+        CONCAT(p.name_, ', ', b.year_publishing) AS publisher_and_year,
+        MAX(DATE_FORMAT(l.loan_date, '%b, %d %Y')) AS loan_date,
```

Comment:
What does MAX() mean here?

Reply:
This line selects the last date of the loan book among all active loans. Using MAX guarantees that if the same book has been borrowed several times, then the last borrowing date will be displayed as a result.

MAX(DATEDIFF(l.due_date, CURDATE())) AS days_until_end_term - Here, MAX is used to determine the maximum number of days until the end of the book return period from all current loans. This is useful if you want to know how many days are left until the very last day when the book must be returned.